qkrehdwns/likelion-assignment-crud/src/main/java/likelion/likelionassignmentcrud/controller/CountryController.java

```diff
@@ -0,0 +1,42 @@
+package likelion.likelionassignmentcrud.controller;
+
+import likelion.likelionassignmentcrud.domain.Country;
+import likelion.likelionassignmentcrud.dto.country.CountryResponseDto;
+import likelion.likelionassignmentcrud.dto.country.CountrySaveDto;
+import likelion.likelionassignmentcrud.service.CountryService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+@Controller
+@RequiredArgsConstructor
+public class CountryController {
+
+    private final CountryService countryService;
+
+    @PostMapping("/country/save")
+    public ResponseEntity<String> saveCountry(@RequestBody CountrySaveDto countrySaveDto) {
+
+        countryService.saveCountry(countrySaveDto);
+
+        return new ResponseEntity<>("사용자 저장!", HttpStatus.CREATED);
```

Comment:
현재 메서드에서는 “사용자 저장!” 이라는 메시지를 반환하고 있어요!
이는 도메인의 의미와 맞지 않기 때문에 "국가 저장!" 등으로 문구를 정확하게 수정해주면 더 좋을 것 같아요 🙂

```suggestion
        return new ResponseEntity<>("국가 저장!", HttpStatus.CREATED);
```